cli/src/commands/commands.ts

```diff
@@ -1,20 +1,34 @@
+import { Compile } from './compile';
 import { Init } from "./init";
 
 /**
  * @description List of command of Subsembly-cli
  */
-export const commands = [{
-    command: 'init [to]',
-    description: 'Initialize a new Subsembly starter project',
-    // @ts-ignore
-    builder: (yargs) => {
-        yargs.positional('to', {
-            describe: 'Directory to initialize new Subsembly project in. Defaults to current directory.',
-            type: 'string'
-        })
+export const commands = [
+    {
+        command: 'init [to]',
+        description: 'Initialize a new Subsembly starter project',
+        // @ts-ignore
+        builder: (yargs) => {
+            yargs.positional('to', {
+                describe: 'Directory to initialize new Subsembly project in. Defaults to current directory.',
+                type: 'string'
+            })
+        },
+        //@ts-ignore
+        handler: async (argv) => {
+            await Init.run(argv.to ? argv.to : '');
+        }
     },
-    //@ts-ignore
-    handler: async (argv) => {
-        await Init.run(argv.to ? argv.to : '');
+    {
+        command: 'compile',
+        description: 'Compile Subsembly project',
```

Comment:
We could have the following description:
`Compiles the Subsembly project placed in the current directory to a WASM bytecode`